src/main.jsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.jsx";
 import "./index.css";
+import "./store";
```

Comment:
Why do we need this store import here?

Reply:
Would be removed... Used for testing if Redux is successfully installed